micropsi_integration_sdk/robot_interface_collection.py

```diff
@@ -20,6 +20,10 @@ def list_robots(self):
     def get_robot_interface(self, robot_model):
         return self.__robots[robot_model]
 
+    def has_internal_ft_support(self, robot_model):
+        return True if robot_model in self.__robots.keys() and\
+            self.__robots[robot_model].has_internal_ft_sensor() else False
```

Comment:
Unnecessary ternary.
`return model in self.__robots and self.__robots[model].has...()`